components/ws-manager/pkg/manager/create.go

```diff
@@ -383,6 +383,17 @@ func (m *Manager) createDefiniteWorkspacePod(startContext *startWorkspaceContext
 	for k, v := range req.Metadata.Annotations {
 		annotations[workspaceAnnotationPrefix+k] = v
 	}
+	if len(startContext.Request.Spec.SshPublicKeys) != 0 {
```

Comment:
it would be good to add cdwp test case for this

Reply:
**C**reate **D**efinite **W**orkspace **P**od
https://github.com/gitpod-io/gitpod/blob/8d077b6eb5ea400b9a8be87d3a50494b335c6b2f/components/ws-manager/README.md#adding-tests-for-create-definite-workspace-pod